pkg/function/rds_functions_test.go

```diff
@@ -106,7 +128,7 @@ func (s *RDSFunctionsTest) TestPrepareCommand(c *check.C) {
 			for db in "${dblist[@]}";
 			  do echo "backing up $db db" && pg_dump $db -C --inserts > /backup/$db.sql;
 			done
-			tar -zc backup | kando location push --profile '%s' --path "${BACKUP_PREFIX}/${BACKUP_ID}" -
+			tar -zc backup | kando location push --profile "%s" --path "${BACKUP_PREFIX}/${BACKUP_ID}" -
```

Comment:
I see that at some places we are providing profile in double quotes `""` and at some place without. Is that intentional?

Reply:
In some tests we just pass `null` string to this function. We can make it more consistent by passing `"\"null\""` instead, I guess.

Reply:
I am good, even if you want to merge this with current state.